charts/argo-cd/templates/argocd-application-controller/statefulset.yaml

```diff
@@ -264,6 +264,8 @@ spec:
       initContainers:
         {{- tpl (toYaml .) $ | nindent 6 }}
       {{- end }}
+      affinity:
+        {{- include "argo-cd.affinity" (dict "context" . "component" .Values.controller) | nindent 8 }}
```

Comment:
This is reason why I'm trying to enforce same structure everywhere as we can use more complex templates that leverage that.